torchao/quantization/qat/api.py

```diff
@@ -256,9 +257,13 @@ class IntXQuantizationAwareTrainingConfig(AOBaseConfig):
     activation_config: Optional[FakeQuantizeConfigBase] = None
     weight_config: Optional[FakeQuantizeConfigBase] = None
 
+    def __post_init__(self):
```

Comment:
I heard typing_extensions.deprecated is more IDE friendly: https://github.com/pytorch/pytorch/pull/153892#discussion_r2097943320

Reply:
hmm I don't think torchao has a dependency on `typing_extensions`?

Reply:
hmm I don't think torchao has a dependency on `typing_extensions`? Not sure if we want to add a new one

Reply:
hmm I don't think torchao has a dependency on `typing_extensions`? Do we want to add a new one for this?

Reply:
I see, OK probably not worth to add deps since it looks like right now there is no deps for torchao